examples/express-graphql-http-functions/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  // Most ts-node options can be specified here using their programmatic names.
+  "ts-node": {
+    "files": true,
+    // It is faster to skip typechecking.
+    // Remove if you want ts-node to do typechecking.
+    "transpileOnly": false
+  },
+  "grats": {
+    "nullableByDefault": false
+  },
+  "compilerOptions": {
+    "lib": ["esnext"],
```

Comment:
Can you add ` "outDir": "dist",` here? Our integration tests for examples assumes it will find the server in `dist/server.js`